lib/algebra.ts

```diff
@@ -1,5 +1,5 @@
 import * as rdfjs from '@rdfjs/types';
-import { Wildcard } from 'sparqljs';
+import { IriTerm, Wildcard } from 'sparqljs-nrt';
```

Comment:
I think we need to figure out the workflow for depending on a fork rather than changing the import name everywhere. This will be inconvenient later.